apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -40,7 +43,8 @@ const SongSelector = Radium(
           >
             {Object.keys(this.props.songData).map(
               (option, i) =>
-                (this.props.filterOff || !this.props.songData[option].pg13) && (
+                // Song should be displayed if it is not pg13 or if the filter is off.
+                (!this.props.filterOn || !this.props.songData[option].pg13) && (
```

Comment:
It seems we could have a helper function which returns the appropriate list of songs, avoiding doing so much business logic in the renderer.  It would then be a great candidate to be unit tested.